### DISCUSSION TITLE
Non-Parametric Mixed ANOVA?

### DISCUSSION THREAD

**Discussion by kirolle** [raphaelvallat/pingouin#433] [Ideas]
Thank you, first, for developing and maintaining this software. I like it! 
Are there any plans to implement mixed variance analysis tools for Non-Parametric data like nparLD for R?

**Comment by raphaelvallat** [comment]
HI @kirolle,

Unfortunately I'm not planning on adding any new features to Pingouin in the near-future — mostly for lack of time.
